package.json

```diff
@@ -48,6 +48,7 @@
     "@types/webpack-env": "1.13.1",
     "ajv": "~5.2.2",
     "gulp": "~4.0.2",
```

Comment:
is this needed?

Reply:
I have another PR coming after this for removing dependencies